ethereum/client/src/lib.rs

```diff
@@ -211,6 +211,16 @@ impl EthApi {
         url: &str,
         message_queue_address: &str,
         private_key: Option<&str>,
+    ) -> Result<EthApi, Error> {
+        Self::new_with_retries(url, message_queue_address, private_key, None, None).await
```

Comment:
`private_key` still exists in memory, implement own keystore: https://github.com/gear-tech/gear/blob/73f5eefac95829e484d618d630c07320272bd8f1/ethexe/ethereum/src/lib.rs#L167

Reply:
Disscused